### DISCUSSION TITLE
info

### DISCUSSION THREAD

**Discussion by giscus** [PickAID/CrychicDoc#89] [Announcements]
# info

一个包含 Minecraft 开发文档的网站。

https://docs.mihono.cn/zh/info/

**Comment by M1hono** [comment]
此处为文档评论区的入口，该区域作为简易的小论坛使用，你可以通过点击头像来去往每个页面对应的comment地址。
目前使用gitcus，根据反馈会考虑使用其他API，但不考虑引入匿名评论。
```md
你可以使用大部分Github支持的md格式
```
请尊重自己与他人，感谢。

---

Here is the entry to the document comment section, serving as a simple forum. You can visit the corresponding comment page for each section by clicking on the avatar.
Currently, we are using Gitcus, and other APIs may be considered based on feedback, but anonymous comments will not be considered.
```md
You can use most Markdown formats supported by GitHub
```
Please respect yourself and others. Thank you.

---
顺便，禁止复读。💢
